mmdet/integration/nncf/compression.py

```diff
@@ -61,6 +64,30 @@ def is_checkpoint_nncf(path):
     except FileNotFoundError:
         return False
 
+def get_nncf_config_from_meta(path, log_dir=None):
+    logger = get_root_logger()
+    checkpoint = torch.load(path)
+    meta = checkpoint.get('meta', {})
+
+    nncf_enable_compression = meta.get('nncf_enable_compression', False)
+    assert nncf_enable_compression, \
+            'get_nncf_config_from_meta should be run for NNCF-compressed checkpoints only'
+
+    config_text = meta['config']
+    with tempfile.NamedTemporaryFile(prefix='config_', suffix='.py', mode='w') as f_tmp:
+        f_tmp.write(config_text)
+        cfg = mmcv.Config.fromfile(f_tmp.name)
+    nncf_config_part = {
+            'nncf_config': cfg.get('nncf_config'),
+            'find_unused_parameters': cfg.get('find_unused_parameters')
+            }
+    if nncf_config_part.get('nncf_config', {}).get('log_dir'):
```

Comment:
looks like `'nncf_config'` is always present in `nncf_config_part`

Reply:
Ok

Reply:
Fixed